content/posts/jellyfin-10.5.md

```diff
@@ -0,0 +1,109 @@
+justify = "center"
++++
+
+What would you think if I tell you that 10.5 is probably the biggest release we've done? Fasten your seat belts and get ready because the numbers are big:
+**Over 200 contributions from the community were merged and over 500 issues were fixed** since 10.4.3 (the latest stable version), both in Jellyfin and Jellyfin Web,
+the interface you access using your browser.
+
+Although we tried hard, we are even unsure if we got covered everything that's new. We hope so. Anyway, you're advised to take some popcorn before continuing. Let's dive in!
+
+We also want to apologise for the late release: We wanted to make 10.5 possible this Christmas, but we always found that something wasn't complete or finished yet.
+We hope you will forgive us (send everyone who contributed a cake if you do, please!) after trying it!
+
+# Key Features
+
+## Let your content shine like never before
+
+If you're like us, you probably want to have your content showcased as beautiful as possible, right? At least, everyone who contributed to the web interface were really comitted into this, so I it seems that, at least, they want to:
+
+#### Jellyfin 10.4.3
+
+<img src="/images/10.5-release/old.png" name="Old item details page" />
+
+#### Jellyfin 10.5
+
+<img src="/images/10.5-release/new.png" name="New item details page" />
+
+*With 10.5, we wanted to reimagine Jellyfin's Web interface*
+
+Posters are now as relevant as the description of the element and backgrounds introduce you to your content. In fact, I thought that I was the jungle when
+I took that screenshot of the *Jungle's Book* wallpaper. Do you feel it like me?
+
+Notice also that the header of the UI has been reworked a little bit in a *slightly* lighter color (instead of the plain color used by the theme)
+
+Shoutout to all the Web team for their unvaluable contributions in mking this happen, specially to [Julien](https://github.com/mrtimscampi) and 
+[grafixeyehero](https://github.com/grafixeyehero).
+
+You can check the redesign in our [demo server](https://demo.jellyfin.org/stable)
+
+## Faster than the sound and cleaner than a surgery room
+
+As you already know, we're still a really young community that want to provide the best self-hosted media server out there. As with every release,
+we tried to clean up and optimize the old code that we inherited when we forked Emby's code. And that's what we have been doing since
+the day we released 10.4.3. A lot of this work will pass unnoticed to many of you, but they're not less important. I'll get to that in a moment.
+
+Let's start with the frontend, as it's what you see everyday in the end. A lot of work has been done to simplify the code and ease the transition
+to more modern frameworks, which will be really useful in the future for making more awesome things with the interface of the clients. For you, end users, this probably is all rubbish because there is nothing you feel that affects you right now. Plain wrong. Just see this video:
```

Comment:
I would rephrase this. Maybe something along the lines: While the benefits are not directly visible in a comparision of screenshots, they already paid off in increased application speed.